EIPS/eip-6927.md

```diff
@@ -0,0 +1,166 @@
+
+## Motivation
+
+The current NFT ecosystem lacks a standardized approach to combine multiple NFTs into a new one. While some projects have implemented custom solutions, these are often ad hoc and project-specific, limiting their interoperability with other NFTs and platforms. This standard aims to create a standardized method for fusing NFTs, enabling users to create new NFTs by combining existing ones according to customizable combination rules. This will foster more incredible innovation and creativity within the NFT space and create new possibilities for NFT use cases.
+
+## Specification
+
+The keywords “MUST,” “MUST NOT,” “REQUIRED,” “SHALL,” “SHALL NOT,” “SHOULD,” “SHOULD NOT,” “RECOMMENDED,” “MAY,” and “OPTIONAL” in this document are to be interpreted as described in RFC 2119.
+
+**Interface**
+
+The following interface extends the existing ERC-721 standard:
+
+```solidity
+// SPDX-License-Identifier: CC0-1.0
+pragma solidity ^0.8.10;
+
+interface IERC6927 {
+    // Events
+
+    /// @dev Emits when a combo is minted.
+    /// @param owner The owner address of the newly minted combo
+    /// @param comboId The newly minted combo identifier
+    event ComboMinted(address indexed owner, uint256 indexed comboId);
+
+    /// @dev Emits when a combo is dismantled.
+    /// @param owner The owner address of the combo
+    /// @param comboId The dismantled combo identifier
+    event ComboDismantled(address indexed owner, uint256 indexed comboId);
+
+    /// @dev Emits when an owner approves another address
+    /// @param spender The approved address
+    /// @param tokenOwner tokenOwner The NFT's owner
+    /// @param tokenAddresses The NFT's collection addresses
+    /// @param tokenIds The NFT identifiers
+    /// @param allowances The allowed number of uses
+    event ComboApproval(
+        address indexed spender,
+        address indexed tokenOwner,
+        address[] tokenAddresses,
+        uint256[] tokenIds,
+        uint256[] allowances
+    );
+
+    // Structs
+
+    /// @dev A ComboFactor defines the requirements for NFTs from a specific collection
+    /// to participate in the combo and the rules that apply.
+    /// A customized combination rule consists of several ComboFactors.
+    /// @param collectionAddress The NFT collection address
+    /// @param minRequired Minimum number of NFTs required from the collection
+    /// @param maxAllowed Maximum number of NFTs allowed from the collection
+    /// @param lockTokens Indicates if tokens from the collection will be locked when minting a combo
+    /// @param maxReuseCount Maximum total reuse count of a token from the collection in
+    /// this combo collection (only applicable for ERC-721 and when lockTokens is false)
+    struct ComboFactor {
+        address collectionAddress;
```

Comment:
You should mention somewhere what standards the `collectionAddress` is allowed to be. I'm guessing it's ERC-721 and ERC-1155 from your Rationale section?